tests/e2e/support/api/token/utils.ts

```diff
@@ -0,0 +1,88 @@
+import { TokenEnvironmentFactory } from '../../environment'
+import { config } from '../../../config'
+import fetch from 'node-fetch'
+import { User } from '../../types'
+const logonUrl = config.backendUrl + '/signin/v1/identifier/_/logon'
```

Comment:
have a new line between imports and vars
```suggestion
import { User } from '../../types'

const logonUrl = config.backendUrl + '/signin/v1/identifier/_/logon'
```